src/config.ts

```diff
@@ -35,7 +35,25 @@ export default {
     { abbr: 'IST', zone: 'Asia/Kolkata' },
   ],
   OPENSOURCE_JOB_CRON_TIME: process.env.OPENSOURCE_JOB_CRON_TIME || '0 14 * * *', // Default time is everyday at 2pm
-  TIPS_DIRECTORY: './tips'
+  TIPS_DIRECTORY: './tips',
+  WORDS: [
+    {
+      check: 'guys',
+      suggestions: [ 'folks', 'people', 'everyone']
+    },
+    {
+      check: 'bros',
+      suggestions: [ 'folks', 'people', 'everyone']
+    },
+    {
+      check: 'bro',
+      suggestions: [ 'folks', 'people', 'everyone']
+    },
+    {
+      check: 'man',
+      suggestions: [ 'folks', 'people', 'everyone']
```

Comment:
Do you think we could make just one array for these suggestions? Since these are the same suggestions but for different checks (like a variable for `inclusiveWordSuggestions` or `genderNeutralSuggestions`). 
Or do you think we'll need to adjust each array differently for each check, in which case this is a better solution?

Reply:
I did think about combining some of these, but I wasn't sure what else we would have. So I thought I would get something working and then see what other words we want to check for, then we can remove duplication.

like you say `check` could become `checks` and be an array - or something similar